app/javascript/components/FlashNotification.vue

```diff
@@ -82,10 +82,10 @@ export default {
 
     classes(propObject, type) {
       let classes = {};
-      if(propObject.hasOwnProperty('base')) {
+      if(Object.prototype.hasOwnProperty.call(propObject, 'base')) {
```

Comment:
why not `Object.hasOwn(propObject, 'base')`?

Reply:
I don't think that will work, this is what was suggested and is another way of calling a class method by using the class itself instead of an instance of it

Reply:
https://dev.to/aman_singh/what-s-the-deal-with-object-prototype-hasownproperty-call-4mbj